src/mainwindow.ui

```diff
@@ -0,0 +1,2174 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MainWindow</class>
+ <widget class="QMainWindow" name="MainWindow">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>809</width>
+    <height>406</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>MainWindow</string>
+  </property>
+  <property name="styleSheet">
```

Comment:
here and below: please undo the stylesheet changes, as it won't work nicely on dark color schemes

we can have one of our designers give us a nice spec to implement in the future. for now, let's keep it simple (and not as pretty)